### DISCUSSION TITLE
Standalone version

### DISCUSSION THREAD

**Discussion by RibShark** [Aurelio92/RVLoader#91] [GC2Wiimote]
It would be great if we could install the IOS patches and whatever else is required for GC2Wiimote without having to install the entire RVLoader stack; there are definite usecases for being able to control Wii games with a GC controller outside of portable Wiis.

**Comment by Bryceshaw06** [comment]
The patches are very heavily integrated into RVLoader/Hiidra and there really is no point for him to release it outside of RVLoader. He did at one point, and it required Neek2o (https://bitbuilt.net/forums/index.php?threads/gc2wiimote-beta.2945/). The codebase is open source, another game launcher (USBLoader/Wiiflow) can add the patches if they want. Until they do, just use RVLoader on your non-portable Wii. It works great! You don't even have to make it autoboot into RVLoader if you don't want to.

**Comment by NikyPSRZ** [comment]
RVLoader can be that standalone version if you just use it to launch games with it.
